komga/build.gradle.kts

```diff
@@ -98,6 +98,8 @@ dependencies {
   runtimeOnly("com.github.gotson:sqlite-jdbc:3.32.3.6")
   jooqGeneratorRuntime("com.github.gotson:sqlite-jdbc:3.32.3.6")
 
+  implementation("org.lz4:lz4-java:1.7.1")
```

Comment:
It seems like Apache Commons Compress, which we already use, has an XXHash implementation: https://commons.apache.org/proper/commons-compress/javadocs/api-release/org/apache/commons/compress/compressors/lz4/XXHash32.html

Could we use that instead and avoid adding one more dependency?

Reply:
I didn't know there was xxHash implementation in Apache commons and it has the same api as the one I'm using. I suppose there's not really a reason to include entire lz4 java library, since it also includes native libraries for different platforms, increasing fat jar size